src/SWP/Bundle/CoreBundle/Controller/ContentPushController.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SWP\Bundle\CoreBundle\Controller;
+
+use Nelmio\ApiDocBundle\Annotation\ApiDoc;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use SWP\Bundle\ContentBundle\Controller\ContentPushController as BaseContentPushController;
+use SWP\Component\Bridge\Model\PackageInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+class ContentPushController extends BaseContentPushController
+{
+    /**
+     * Receives HTTP Push Request's payload which is then processed by the pipeline.
+     *
+     * @ApiDoc(
+     *     resource=true,
+     *     description="Adds a new content from HTTP Push",
+     *     statusCodes={
+     *         201="Returned on successful post."
+     *     }
+     * )
+     * @Route("/api/{version}/content/push", options={"expose"=true}, defaults={"version"="v1"}, name="swp_api_content_push")
+     * @Method("POST")
+     */
+    public function pushContentAction(Request $request)
```

Comment:
because I need to use multitenancy code to disable tenantable filter in order to check if an article already exists and I didn't include whole multitenancy to content bundle.

If an article was pushed already there is no tenant defined for it so if you update the same article it would then return `null` when chcking for article existence because tenantable filter will append `tenant_code` which doesn't exist.